### ISSUE TITLE
Preprocessors benchmark

### ISSUE THREAD

**Issue by rhiever** [rhiever/sklearn-benchmarks#38]
No major parameters:

http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.Binarizer.html#sklearn.preprocessing.Binarizer

http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MaxAbsScaler.html#sklearn.preprocessing.MaxAbsScaler

http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html#sklearn.preprocessing.MinMaxScaler

http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html#sklearn.preprocessing.StandardScaler

http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.RobustScaler.html#sklearn.preprocessing.RobustScaler

http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.PolynomialFeatures.html#sklearn.preprocessing.PolynomialFeatures

Need to be parameterized:

http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.Normalizer.html

http://scikit-learn.org/stable/modules/generated/sklearn.decomposition.FastICA.html

http://scikit-learn.org/stable/modules/generated/sklearn.decomposition.PCA.html

http://scikit-learn.org/stable/modules/generated/sklearn.kernel_approximation.RBFSampler.html

http://scikit-learn.org/stable/modules/generated/sklearn.kernel_approximation.Nystroem.html

http://scikit-learn.org/stable/modules/generated/sklearn.cluster.FeatureAgglomeration.html

http://scikit-learn.org/stable/modules/generated/sklearn.feature_selection.SelectFwe.html

http://scikit-learn.org/stable/modules/generated/sklearn.feature_selection.SelectPercentile.html

http://scikit-learn.org/stable/modules/generated/sklearn.feature_selection.VarianceThreshold.html

http://scikit-learn.org/stable/modules/generated/sklearn.feature_selection.SelectFromModel.html

http://scikit-learn.org/stable/modules/generated/sklearn.feature_selection.RFE.html
